src/balatro_gym/cards/interfaces.py

```diff
@@ -204,7 +205,12 @@ def create_tarot(self) -> bool:
         return True
 
 
-class PlayingCard(HasChips):
+@dataclasses.dataclass
+class Card(Protocol):
+    cash_value: int = 1
+
+
+class PlayingCard(HasChips, Card):
```

Comment:
Does a playing card have cash value outside of the `GoldCard` enhancement?

Also, there is an existing `HasMoney` mixin (which is poorly named). I'm wondering if we can expand that, or reform it to capture what you needed here.

Reply:
When playing cards are bought in the shop (which can happen when you buy the specific voucher), they have a cash value. It's mostly to capture the price in the shop rather than how much money a card can earn you (like with the gold enhancement). The goal of the `Card` protocol is so we have a common class to represent in the shop state where we have a sequence of cards that can be bought, jokers, tarot cards, planet cards, etc. Also, conceptually it makes sense that all cards would use a common for now the only common property is they all have a cash value to purchase them.

Reply:
Hm I see, that makes sense to me. In that case, what do you think about naming it to `HasCost` rather than `Card`?